### PR TITLE
feat(components): adding new colors and style to action blip table

### PR DESCRIPTION
**Pull request by nicolasamaduro**
adding new colors cloud-burst and athens-gray
adding row hover to table with action
adding style to action icon
adding $u to global variables

feature/146441-adding-action-to-table

### CODE REVIEW

File: src/scss/components/_table.scss [hunk 1]
```diff
@@ -105,6 +110,19 @@
       background-color: $bp-color-whisper;
     }
 
+    .hasTableAction:hover{
+      background-color: $bp-table_tr-hover;
+    }
+  
+    #td-action-icon{
```
Comment (liviaalmeida) [inline src/scss/components/_table.scss]: Don't use ```#```

Comment (nicolasamaduro) [reply src/scss/components/_table.scss]: why?

File: src/scss/utilities/_color.scss [hunk 2]
```diff
@@ -46,6 +47,7 @@
 @include register-base-color('wave', $bp-color-wave);
 @include register-base-color('sky', $bp-color-sky);
 @include register-base-color('offwhite', $bp-color-offwhite);
+@include register-base-color('athens-gray', $bp-color-athens-gray);
```
Comment (liviaalmeida) [inline src/scss/utilities/_color.scss]: Register new color in button colors as well

File: src/scss/utilities/_color.scss [hunk 3]
```diff
@@ -33,6 +33,7 @@
 @include register-base-color('console', $bp-color-console);
 @include register-base-color('obsidian', $bp-color-obsidian);
 @include register-base-color('onix', $bp-color-onix);
+@include register-base-color('cloud-burst', $bp-color-cloud-burst);
```
Comment (liviaalmeida) [inline src/scss/utilities/_color.scss]: Register new color in button colors as well

**Pull request merged**